### PR TITLE
Saturday

### LINKED ISSUE #14: Create Seed File
Create seed file with users/clients/reviews

### PR DESCRIPTION
**Pull request by ahenzler**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] SimpleCov at 100%
- [ ] RuboCop complaints resolved

closes #14

### CODE REVIEW

File: app/graphql/types/client_type.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module Types
+  class ClientType < Types::BaseObject
+    field :id, ID, null: false
+    field :email, String, null: false
+  end
+end
```
Comment (LawrenceWhalen) [inline app/graphql/types/client_type.rb:6]: I don't know why but this feels like we can let users store some more preferences as extensions.

Comment (LawrenceWhalen) [reply app/graphql/types/client_type.rb:6]: Just feels sparse, but that might be good.

File: app/graphql/types/query_type.rb [hunk 2]
```diff
@@ -18,5 +19,37 @@ def all_users
...
+  # REVIEWS
+  field :all_reviews, [Types::ReviewType], null: false
+    def all_reviews
+      Review.all
+    end
+
+  field :reviews_by_rating_high_to_low, [Types::ReviewType], null: false
+    def reviews_by_rating_high_to_low
+      Review.order(rating: :desc)
+    end
```
Comment (LawrenceWhalen) [inline app/graphql/types/query_type.rb:45]: Want to research this more, but it I am so glad you guys got this to work.

**Review by Aferencz1987 — APPROVED** [review]

File: app/graphql/types/review_type.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    field :size, Float, null: false
+    field :payment, Integer, null: false
+    field :extended_body, String, null: false
+    field :kindness, Integer, null: false
+    field :vibe, Integer, null: false
+    field :date_again, Boolean, null: false
+    field :safety_meter, Integer, null: false
+    field :gender, String, null: false
+  end
+end
```
Comment (LawrenceWhalen) [inline app/graphql/types/review_type.rb:18]: Remove null false

File: app/models/client.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class Client < ApplicationRecord
+  has_many :reviews, dependent: :destroy
```
Comment (LawrenceWhalen) [inline app/models/client.rb:2]: Thank you for adding this, orphaned would be annoying.

File: db/migrate/20210829162034_add_collumns_to_review.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  def change
+    add_column :reviews, :title, :string
+    add_column :reviews, :size, :float
+    add_column :reviews, :payment, :integer
+    add_column :reviews, :extended_body, :string
+    add_column :reviews, :kindness, :integer
+    add_column :reviews, :vibe, :integer
+    add_column :reviews, :date_again, :boolean
+    add_column :reviews, :safety_meter, :integer
+    add_column :reviews, :gender, :string
```
Comment (LawrenceWhalen) [inline db/migrate/20210829162034_add_collumns_to_review.rb:11]: Well done Alex!

**Review by LawrenceWhalen — APPROVED** [review]
Wonderful work everyone, the foundation is coming together quickly!

**Pull request merged**